<launch>
    <node pkg="joy" exec="joy_node" name="joy_node"/>
    <node pkg="behavior" exec="takeoff" name="TakeOff"/>
    <node pkg="behavior" exec="land" name="Land"/>
    <node pkg="behavior" exec="hover" name="Hover"/>
    <node pkg="behavior" exec="move_forward" name="MoveForward"/>
    <node pkg="behavior" exec="move_left" name="MoveLeft"/>
    <node pkg="low_level_controller" exec="speed_controller" name="speed_controller"/>
    <node pkg="twist2pic" exec="twist2pic" name="twist2pic"/>
    <node pkg="behavior" exec="status_viewer" name="status_viewer"/>
    <node pkg="behavior" exec="joy_teleop" name="joy_teleop"/>
    <node pkg="joy" exec="joy_node" name="joy_node"/>
    <node pkg="behavior" exec="command" name="command"/>
    

</launch>
